assets/app/features/upgrade/steps.factory.spec.js

```diff
@@ -50,13 +50,541 @@ describe('Stepes Factory', function () {
             active: false,
             finished: false
         }
-    ];
+        ],
+        initialStatusData = {
```

Comment:
There are 13 exact same structures hardcoded here with just different values occupying 500+ lines of code. There must be a more efficient and maintainable way to generate this.